docs/playlist/rap.md

```diff
@@ -8,3 +8,4 @@ title: Rap
 | Playlist Name | Playlist Link | Platform | Playlist Author Name/Contributor Name | Note | Subgenre |
 | -------------- |-------------- | -------- | ------- | ------- | -------- |
 | It's a Dad Thing 2 | [YouTube](https://www.youtube.com/watch?v=hcRoc3gqu-Q&list=OLAK5uy_nzTAQaRiE0CEth1jUAbw1s1bTcO_kgnoY) | Youtube | [saykojigor](https://www.youtube.com/user/saykojigor) | - |
+| 3 | [Spotify](https://open.spotify.com/playlist/5gqzMovSFP4j7GYzepEImC?si=2d5b0b7413a7487a) | Spotify | TheAnton205 | - | - |
```

Comment:
Please sort alphabetically and also sort numeric first, refer to https://awesome-music-playlist.vercel.app/contribution.html#add-new-playlist

Reply:
Fixed, my apologies